website/pages/docs/configuration/storage/raft.mdx

```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
   snapshot. Servers may take longer to recover from crashes or failover if this
   is increased significantly as more logs will need to be replayed.
 
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
+other nodes that they could join using this config. When one of the nodes is
+initialized, it becomes the leader and all the other nodes will join the
+initialized node to form the cluster. See the section below that describes the
+parameters accepted by the `retry_join` stanza.
```

Comment:
Could you add a few spaces to these to they line up like the fields above?

Reply:
Done.